### ISSUE TITLE
测试表格“复制: test by jian”： 填写多项选择（其他），提交后，没有记录结果

### ISSUE THREAD

**Issue by jianhe7** [yannonghuang/shinshin#66]
<img width="290" alt="Screen Shot 2022-06-19 at 4 21 31 PM" src="https://user-images.githubusercontent.com/11562782/174504390-caea951e-a8b0-4a76-ade4-2731be67ccc8.png">
<img width="332" alt="Screen Shot 2022-06-19 at 4 21 58 PM" src="https://user-images.githubusercontent.com/11562782/174504395-c4ca5d17-da41-4896-ba59-de8c5b7857b2.png">

**Comment by yannonghuang** [comment]
项目申请表（项目征集表）不记录填表结果，填表结果记录在项目申请中

**Comment by jianhe7** [comment]
其他问题有记录，只是多项选择没有。插图里选择的 “女”记录下了，但是，科目下面选择的结果没有记录。这个可能是表格插件有问题？我测试多项里如果用OTHER。

**Comment by yannonghuang** [comment]
It seems working fine without "required" checked. Suggested not to use "required". It does not do any good anyway?

**Comment by jianhe7** [comment]
Yes, looks like "required" doesn't work at all. If I didn't fill the required question, it doesn't alert the problem either. Could you check if the form plugin has any updates to fix this problem? Do you know where you may submit this bug to the developer of this plugin?

**Comment by yannonghuang** [comment]
This is an open bug on the formbuilder's github site. It's been open for awhile.
